Servers/patching-as-a-service.md

```diff
@@ -128,7 +129,7 @@ The process initiated by the script package may include several, automated reboo
 
 2\. Execute Action
 
-Navigate to the group and select "execute package" from the [action drop-down](../Servers/using-group-tasks-to-install-software-and-run-scripts-on-groups.md) . The name of the package to search for is in the table above.
+Navigate to the group and select "execute package" from the [action drop-down](../servers/using-group-tasks-to-install-software-and-run-scripts-on-groups.md) . The name of the package to search for is in the table above.
```

Comment:
@aimeemudd this change introduces a broken link. You can see the error here: https://travis-ci.org/CenturyLinkCloud/PublicKB/builds/75327144

Folder names are case sensitive, so `servers` should be `Servers`.